src/FormBuilder/resources/js/form-builder/src/components/sidebar/panels/FormPrepublishPanel.tsx

```diff
@@ -0,0 +1,152 @@
+import {CopyIcon} from '@givewp/form-builder/components/icons';
+import FormSummarySettings from '@givewp/form-builder/settings/form-summary';
+import {getWindowData} from '@givewp/form-builder/common';
+
+interface FormPrepublishPanelProps {
+    isSaving: boolean;
+    isPublished: boolean;
+    handleSave: MouseEventHandler<HTMLButtonElement>;
+    handleClose: MouseEventHandler<HTMLButtonElement>;
+}
+
+export default function FormPrepublishPanel
+({
+     isSaving,
+     isPublished,
+     handleSave,
+     handleClose
+ }: FormPrepublishPanelProps) {
+
+    const {
+        settings: {formTitle, formStatus, newFormStatus},
+    } = useFormState();
+    const dispatch = useFormStateDispatch();
+    const {
+        formPage: {permalink},
+    } = getWindowData();
+
+    const isPrivate = () => {
+        if (newFormStatus) {
+            return 'private' === newFormStatus;
+        }
+
+        return 'private' === formStatus;
+    }
+
+    const permalinkField: RefObject<HTMLInputElement> = createRef();
+
+    return createPortal(
+        <div
+            className={cx('givewp-next-gen-prepublish-panel', {'givewp-next-gen-prepublish-panel__animate': !isPublished})}>
+            {isSaving ? (
+                <>
+                    <div className="givewp-next-gen-prepublish-panel__header">
+                        <div className="givewp-next-gen-prepublish-panel__header-actions">
+                            <Button
+                                variant="primary"
+                                isBusy={true}
+                                isPressed={true}
+                            >
+                                {__('Publishing', 'give')}
+                            </Button>
+                        </div>
+                    </div>
+                    <div className="givewp-next-gen-prepublish-panel__spinner">
+                        <Spinner />
+                    </div>
+                </>
+            ) : (
+                <>
+                    {isPublished ? (
+                        <>
+                            <div className="givewp-next-gen-prepublish-panel__header">
+                                <div className="givewp-next-gen-prepublish-panel__header-actions">
+                                    <Button
+                                        variant="secondary"
+                                        onClick={handleClose}
+                                    >
+                                        {__('Close panel', 'give')}
+                                    </Button>
+                                </div>
+                            </div>
+
+                            <div className="givewp-next-gen-prepublish-panel__content_publish">
+                                <strong>
+                                    <a href={permalink}
+                                       target="_blank">{formTitle}
+                                    </a> {__('is now live', 'give')}.
```

Comment:
For translations, we need to include a placeholder for reference, otherwise the translayfile has no context of "what" is now live.

Also, we should probably use Interweave to include the a tag mark up in the placeholder.

Reply:
Resolved https://github.com/impress-org/givewp/pull/7050/commits/4b14bad6565524f4854f62c0661914796736924c